<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Formatos</class>
 <widget class="QDialog" name="Formatos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEditFiltro"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;---Filtro</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButtonModificar">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Modificar</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QPushButton" name="pushButtonnadir">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>AÃ±adir</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pushButtonCerrar">
     <property name="text">
      <string>Cerrar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonCerrar</sender>
   <signal>clicked()</signal>
   <receiver>Formatos</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
